src/css/proposal.css

```
@@ -1 +1,403 @@
-/* proposal styles */
\ No newline at end of file
+/* proposal styles */
+.proposal-section .container {
+    padding-bottom: 0;
+    padding-left: 20px;
+    background: var(--primary-bg-color);
+    border-radius: 16px;
+    height: 737px;
+    width: 335px;
```

We shouldn't override styles in the `container` class since it's a general class used throughout the site. It's better to create an additional class and add the necessary styles to it. For example, `proposal-container`.
```suggestion
    min-height: 737px;
    min-width: 335px;
```

---

created